### DISCUSSION TITLE
does vcpkg have a telemetry data?

### DISCUSSION THREAD

**Discussion by ilya-lavrenov** [microsoft/vcpkg#33593] [General]
It would be very convenient if `vcpkg` provides telemetry data to understand what ports are popular and so on.
It helps to understand for ports maintainers whether the port is popular and pay extra attention to it.

**Comment by autoantwort** [comment]
Microsoft have them, but they are not public

**Comment by ilya-lavrenov** [comment]
Any reason why they are private?

**Reply by autoantwort** [reply]
I guess because they are send to an internal monitoring tool that is not public and it needs work time to make it public and that costs money

**Reply by ilya-lavrenov** [reply]
Money is not a reason to have private statistic.
